index.html

```diff
@@ -27,7 +27,7 @@
         <meta charset="UTF-8">
 
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.6.1/dat.gui.min.js"></script>
+        <script src="node_modules/dat.gui/build/dat.gui.min.js"></script>
```

Comment:
I'm not sure what you mean. I did change `autodeploy.sh` right?

> Keep old index.html in README.md

I didn't change `README.md`. 

As for a dev demonstration, I thought we could let it for people doing `npm start`. They would be the one interested in it anyway.